src/main/scala/org/graphframes/lib/ParallelPersonalizedPageRank.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.graphframes.lib
+
+import org.apache.spark.graphx.{lib => graphxlib}
+import org.graphframes.{GraphFrame, Logging}
+
+class ParallelPersonalizedPageRank private[graphframes] (
+  private val graph: GraphFrame) extends Arguments {
+
+  private var resetProb: Option[Double] = Some(0.15)
+  private var numIter: Option[Int] = Some(10)
```

Comment:
It looks like the [runParallelPersonalizedPageRank](http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.graphx.lib.PageRank$@runParallelPersonalizedPageRank[VD,ED](graph:org.apache.spark.graphx.Graph[VD,ED],numIter:Int,resetProb:Double,sources:Array[org.apache.spark.graphx.VertexId])(implicitevidence$5:scala.reflect.ClassTag[VD],implicitevidence$6:scala.reflect.ClassTag[ED]):org.apache.spark.graphx.Graph[org.apache.spark.ml.linalg.Vector,Double]) function is a bit different from the standard `PageRank` interface, where the former only exposes the function for fixed number of iterations (but not the tolerance based version).